controllers/volumereplicationgroup_controller.go

```diff
@@ -377,12 +377,16 @@ const (
 	pvcVRFinalizerProtected = "volumereplicationgroups.ramendr.openshift.io/pvc-vr-protection"
 	pvcInUse                = "kubernetes.io/pvc-protection"
 
-	// Annotations
+	// pvc Annotations
 	pvcVRAnnotationProtectedKey   = "volumereplicationgroups.ramendr.openshift.io/vr-protected"
 	pvcVRAnnotationProtectedValue = "protected"
-	pvVRAnnotationRetentionKey    = "volumereplicationgroups.ramendr.openshift.io/vr-retained"
-	pvVRAnnotationRetentionValue  = "retained"
-	PVRestoreAnnotation           = "volumereplicationgroups.ramendr.openshift.io/ramen-restore"
+	pvcVRAnnotationOwnerKey       = "volumereplicationgroups.ramendr.openshift.io/vrg-name"
+
+	// pv Annotations
+	pvVRAnnotationRetentionKey   = "volumereplicationgroups.ramendr.openshift.io/vr-retained"
+	pvVRAnnotationRetentionValue = "retained"
+	PVRestoreAnnotation          = "volumereplicationgroups.ramendr.openshift.io/ramen-restore"
+	pvVRAnnotationOwnerKey       = "volumereplicationgroups.ramendr.openshift.io/vrg-name"
```

Comment:
PV owner is needed for MetroDR? As ownership of PVC is enough to decide not to protect the PVC, the PV should not matter at that point is the assumption.

Reply:
This can be removed. I don't think it is required.